components/locid/src/extensions/unicode/keywords.rs

```diff
@@ -49,27 +49,25 @@ impl Keywords {
     /// ```
     #[inline]
     pub const fn new() -> Self {
-        Self(Vec::new())
+        Self(LiteMap::new())
     }
 
-    /// A constructor which takes a pre-sorted list of `(`[`Key`]`, `[`Value`]`)` tuples.
-    ///
+    /// Returns `true` if there are no keywords.
     ///
     /// # Examples
     ///
     /// ```
-    /// use icu::locid::extensions::unicode::{Keywords, Key, Value};
+    /// use icu::locid::Locale;
+    /// use icu::locid::extensions::unicode::Keywords;
     ///
-    /// let key: Key = "ca".parse()
-    ///     .expect("Failed to parse a Key.");
-    /// let value: Value = "buddhist".parse()
-    ///     .expect("Failed to parse a Value.");
-    /// let keywords = Keywords::from_vec_unchecked(vec![(key, value)]);
+    /// let loc1 = Locale::from_bytes(b"und-t-h0-hybrid").unwrap();
+    /// let loc2 = Locale::from_bytes(b"und-u-ca-buddhist").unwrap();
```

Comment:
`nit`: here and elsewhere, can you use `expect` in place of `unwrap` in examples?

Reply:
As mentioned in the other PR, we can fix this over the `icu_locid` crate in a follow-up.

We should discuss whether we want this to be a policy for all ICU4X docs. I'm not sure.